docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,37 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+Ledger API
+~~~~~~~~~~
+- **BREAKING** Removed the unused field :ref:`com.digitalasset.ledger.api.v1.ExercisedEvent` from :ref:`com.digitalasset.ledger.api.v1.Event`.
+  ``Event`` is only used in :ref:`com.digitalasset.ledger.api.v1.Transaction`, which in turn by definition never contains exercised events (only created and archived events): `#960 <https://github.com/digital-asset/daml/issues/960>`_
+
+  This change is *backwards compatible on the transport level*, meaning:
+
+  - new versions of ledger language bindings will work with previous versions of the sandbox, as the field was never populated,
+  - previous versions of the ledger language bindings will work with new versions of the sandbox, as the field was removed without any change in observable behaviour.
+
+  Migration:
+
+  - If you check for the presence of ``ExercisedEvent`` when handling a :ref:`com.digitalasset.ledger.api.v1.Transaction`, you have to remove this code now.
+
+Java Bindings
+~~~~~~~~~~~~~
+- **BREAKING** Reflect breaking change of Ledger API in the event class hierarchy: `#960 <https://github.com/digital-asset/daml/issues/960>`_
+
+  - ``data.Event`` has been changed from an abstract class to an interface, representing events in a flat transaction.
```

Comment:
```suggestion
  - Changed ``data.Event`` from an abstract class to an interface, representing events in a flat transaction.
```

Reply:
Yikes, I keep on being a passive-offender :) Thank you.